NEWS.md

```diff
@@ -4,6 +4,9 @@
 * `install_github()` now uses credentials from the git credential store,
   if `GITHUB_PAT` and `GITHUB_TOKEN` are not set.
 * Remotes field accepts explicit package names: `<pkgname>=<type>::<username>/<repo>` (#719, @heavywatal).
+* Add `remote_precedence` parameter to `dev_package_deps()` and `combine_remote_deps()` that allows
+  user to choose whether remote sources should have a priority over cran sources. Parameter defaults
+  to TRUE for backward compatibility reasons. (#732, @maksymiuks)
```

Comment:
It is not "just" for backward compatibility, but rather because that's the usual case. So I would just remove this last sentence.

Reply:
right, done!